packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-subgraph/index.esm.t

```diff
@@ -0,0 +1,43 @@
+---
+to: "<%= rootDir %>/<%= h.changeCase.param(name) %>/index.ts"
+force: true
+---
+import { Subgraph } from "@powerhousedao/reactor-api";
+import { Knex } from "knex";
+import { gql } from "graphql-tag";
```

Comment:
This will require graphql-tag and graphql as dependencies in the boilerplate. Maybe the typedefs could just be defined as a string here?
The subgraph manager would parse it into typedefs.

Reply:
I would argue, that if you decide to develop a subgraph you might accept that you have a graphql dependency... gql is a nice helper and supported by vscode graphql extension with syntax highlighting. if its just a string the dev dont get syntax higlighting :(